heros/hero1.html.slim

```diff
@@ -0,0 +1 @@
+= partial('hero1.html.slim');
```

Comment:
Why do you need this indirection? Can't you just put the contents of the partial inline?